pages/linux/zramctl.md

```diff
@@ -0,0 +1,23 @@
+# zramctl
+
+> Setup and control zram devices.
+
+- Check if zram is enabled:
+
+`lsmod | grep -i zram`
+
+- Enable zram with 2 devices:
+
+`sudo modprobe zram num_devices={{2}}`
+
+- Initialise the next free zram device to a 2GB virtual drive:
+
+`sudo zramctl --find --streams {{4}} --size {{2GB}} --algorithm lz4`
```

Comment:
I will take a stab at it when I have some time. Now that I have a clear idea as to what the concepts are.